lib/camp.js

```diff
@@ -773,6 +774,38 @@ function pathLikeUnit(serverMethod) {
 var pathUnit = pathLikeUnit('path');
 var notFoundUnit = pathLikeUnit('notFound');
 
+// HTTP method unit (express-like, e.g. `camp.post('/', fn)`).
+function methodUnit (server) {
```

Comment:
I agree it's better to support method-specific handlers in `pathUnit` directly, in order to preserve routing order.

Express only has single-method handlers (e.g. `app.get(path, (req, res) => {})`, `app.post(path, (req, res) => {})`, etc.) and one every-method handler that acts like a middleware (e.g. `app.all(path, (req, res, next) => {})`, so that you can let subsequent handlers catch your requests if you end up not handling them). I guess that's ok for most use cases. Do we really need N-method handlers that can set any number of methods?